### ISSUE TITLE
comment-spacings should allow //<Tab> and /*

### ISSUE THREAD

**Issue by powerman** [mgechev/revive#788]
**Describe the bug**
Rule comment-spacings does not allow neither `//<Tab>` nor `/*`.
https://go.dev/blog/godoc refer https://go.dev/src/pkg/encoding/gob/doc.go as an example of doc formatting which uses `//<Tab>` for indenting source code example.

**To Reproduce**
1. I updated revive `go install github.com/mgechev/revive@latest`
2. I run it with the following flags & configuration file:

```shell
revive -config <(echo "[rule.comment-spacings]") encoding/gob
```

**Expected behavior**
No errors should be detected by the rule.

**Logs**
```
/usr/lib/go/src/encoding/gob/doc.go:5:1: no space between comment delimiter and comment text
/usr/lib/go/src/encoding/gob/doc.go:290:1: no space between comment delimiter and comment text
/usr/lib/go/src/encoding/gob/doc.go:338:1: no space between comment delimiter and comment text
/usr/lib/go/src/encoding/gob/decoder.go:138:1: no space between comment delimiter and comment text
/usr/lib/go/src/encoding/gob/codec_test.go:1353:2: no space between comment delimiter and comment text
/usr/lib/go/src/encoding/gob/encoder_test.go:164:2: no space between comment delimiter and comment text
/usr/lib/go/src/encoding/gob/encoder_test.go:165:2: no space between comment delimiter and comment text
/usr/lib/go/src/encoding/gob/encoder_test.go:179:2: no space between comment delimiter and comment text
/usr/lib/go/src/encoding/gob/encoder_test.go:604:2: no space between comment delimiter and comment text
/usr/lib/go/src/encoding/gob/encoder_test.go:605:2: no space between comment delimiter and comment text
```

**Desktop (please complete the following information):**
 - OS: Gentoo Linux
 - go 1.20
